components/content/Text/Text.css

```diff
@@ -0,0 +1,188 @@
+/* Font */
+
+.diamond-text-font-body {
+  font-size: var(--diamond-font-size-base);
+  font-weight: var(--diamond-font-weight-base);
+  line-height: var(--diamond-font-line-height);
+  text-wrap: pretty;
+}
+
+.diamond-text-font-heading,
+.diamond-text-font-h1,
+.diamond-text-font-h2,
+.diamond-text-font-h3,
+.diamond-text-font-h4 {
+  font-weight: var(--diamond-font-weight-bold);
+  line-height: var(--diamond-font-line-height-sm);
```

Comment:
We should have `--diamond-font-line-height-heading`, otherwise people won't have a way to change it for headings only.

It can still be set to line-height-sm `--diamond-font-line-height-heading: var(--diamond-font-line-height-sm);`